worlds/hades/Rules.py

```diff
@@ -0,0 +1,420 @@
+from BaseClasses import MultiWorld, Item, ItemClassification
+from .Items import item_table_pacts, item_table_weapons, item_table_keepsake, items_table_fates_completion, HadesItem
+from ..AutoWorld import LogicMixin
+from ..generic.Rules import set_rule, add_rule, add_item_rule
+from Utils import visualize_regions
+from .Locations import location_weapons_subfixes
+
+class HadesLogic(LogicMixin):
+    #This is not working as expected
```

Comment:
If this is still not working as expected, here's a reminder to fix it (or remove the note)

Reply:
It is, thankfully. Again, some of those things that have been there since forever and grow (badly) accustomed to it.